Kitodo/src/main/java/org/kitodo/production/services/dataeditor/DataEditorService.java

```diff
@@ -242,7 +242,7 @@ public static List<SelectItem> getAddableMetadataForMediaUnit(DataEditorForm dat
      */
     public static StructuralElementViewInterface getStructuralElementView(DataEditorForm dataEditor) {
         Optional<LogicalDivision> selectedStructure = dataEditor.getSelectedStructure();
-        if (selectedStructure.isPresent()) {
+        if (Objects.nonNull(selectedStructure) && selectedStructure.isPresent()) {
```

Comment:
The null check could be avoided if the `getSelectedStructure()` method is changed in the already mentioned way (`return null` -> `return Optional.empty()`.